R/locale.R

```diff
@@ -1,84 +1,26 @@
-overrideWindowsLocale <- function(locale) {
```

Comment:
Might need to restore, but this seems rather ad hoc, and I can't see the motivating issue at https://github.com/rstudio/shinyapps/commit/b5f9f7cd5a5d9a5cafabd2ef7e234cfa643a663e

Reply:
Here's a stackoverflow mentioning the same:
https://stackoverflow.com/questions/30163941/r-fail-to-deploy-on-shinyapps-io-unsupported-locale-greek-el-el

Reply:
Original issue: https://groups.google.com/g/shinyapps-users/c/nUgepQwRcO4/m/na3hkd6Va5UJ?utm_medium=email&utm_source=footer&pli=1

Reply:
Looks like that's the same person; so it's not two independent instances.

Reply:
I think this approach is a big improvement, but I'm not sure how to confirm that Windows actually returns the correct language code with this method. The Greek case could have been a bug, but is it fixed?

Reply:
Given that the previous fix was applied for a single person who had a problem in 2018, I'd prefer to err on the side of keeping the code simple. However, if you feel strongly about it, I'll add it back in.

Reply:
That’s fine with me.

Reply:
I installed Greek on Windows 10, the version without a region, and the registry has `el-GR`, so this PR is probably OK.